### ISSUE TITLE
office-ui-fabric-react-bundle 404 Not found

### ISSUE THREAD

**Issue by maleman82** [SharePoint/sp-dev-docs#555]
Thank you for reporting an issue or suggesting an enhancement. We appreciate your feedback - to help the team to understand your needs, please complete the below template to ensure we have the necessary details to assist you. 

#### Category
- [x ] Question
- [ ] Typo
- [x] Bug
- [ ] Enhancement / Suggestion
- [ ] Additional article idea

#### Expected or Desired Behavior
Initial install of SharePoint framework and generation of shell program "hello world" should display in my browser as in the tutorial.

#### Observed Behavior
Everything seems to start fine using gulp serve, but it displays a blank page in the browser. In the console I am receiving the following error: 404 not found - https://localhost:4321/node_modules/@microsoft/office-ui-fabric-react-bundle/02a01e42-69ab-403d-8a16-acd128661f8e. 

#### Steps to Reproduce
I have tried creating a new folder and starting over, but I receive the same error.

Thanks for your assistance!

**Comment by manishgarg1** [comment]
Hi @maleman82, can you please share the contens of the "node_modules\@microsoft\office-ui-fabric-react-bundle" folder, and the "dist" folder inside that.

**Comment by maleman82** [comment]
Thank you Manish for looking into this!

[office-ui-fabric-react-bundle.zip](https://github.com/SharePoint/sp-dev-docs/files/948596/office-ui-fabric-react-bundle.zip)

**Comment by maleman82** [comment]
Any progress yet on this issue?

**Comment by maleman82** [comment]
Finally after numerous attempts, patches, tweaks, I believe I have found the problem. Apparently, I did not have the right or matching versions of the generator-sharepoint, gulp-core-build-typescript, and the yo Client-side generator. Evidently I had a problem when I originally updated the generator-sharepoint. It had not actually updated like I thought it had.

So far I am able to create a new project and view it in the browser window!
